cloudinit/sources/DataSourceVMware.py

```diff
@@ -116,11 +121,19 @@ class DataSourceVMware(sources.DataSource):
             Network Config Version 2 - http://bit.ly/cloudinit-net-conf-v2
 
         For example, CentOS 7's official cloud-init package is version
-        0.7.9 and does not support Network Config Version 2. However,
-        this datasource still supports supplying Network Config Version 2
-        data as long as the Linux distro's cloud-init package is new
-        enough to parse the data.
-
+        0.7.9 and does not support Network Config Version 2.
+
+        imc transport:
```

Comment:
Please indent the descriptions under the transport type. Otherwise it is difficult to parse.

Reply:
Updated, thanks.